src/utils.cpp

```diff
@@ -1069,6 +1069,11 @@ std::string utils::make_title(const std::string& const_url) {
 	if (title.at(0)>= 'a' && title.at(0)<= 'z') {
 		title[0] -= 'a' - 'A';
 	}
+	//strip .htm or .html from title
+	size_t pos = title.find(".html",0);
```

Comment:
I'd rather limit this to the end of the URL. We probably don't want to strip ".html" from the middle of the title.

I also don't mind seeing this re-written with a regex and extended to cover `.php` and `.aspx` as these are quite common; but that's up to you.